src/pymovements/events/ivt.py

```diff
@@ -0,0 +1,86 @@
+from __future__ import annotations
+
+from typing import Optional
+import numpy as np
+
+from pymovements.transforms import vnorm
+from pymovements.events import Fixation
+
+
+def ivt(
```

Comment:
should we include `eye`as an argument or rather decide which eye to use via input? I'm unsure, maybe include @dkrako

Reply:
we will stay monocular for the current milestone and add binocular functionality later on.